src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -39,27 +40,121 @@
 
 namespace {
 
+const QString kRekordboxLibraryTable = QStringLiteral("rekordbox_library");
+const QString kRekordboxPlaylistsTable = QStringLiteral("rekordbox_playlists");
+const QString kRekordboxPlaylistTracksTable = QStringLiteral("rekordbox_playlist_tracks");
+
 const QString kPdbPath = QStringLiteral("PIONEER/rekordbox/export.pdb");
 const QString kPLaylistPathDelimiter = QStringLiteral("-->");
 const double kLongestPosition = 999999999.0;
 
-void clearTable(QSqlDatabase& database, QString tableName) {
+enum class TrackColor {
+    TrackColorPink = 1,
+    TrackColorRed,
+    TrackColorOrange,
+    TrackColorYellow,
+    TrackColorGreen,
+    TrackColorAqua,
+    TrackColorBlue,
+    TrackColorPurple
+};
+
+constexpr mixxx::RgbColor kPinkTrackColor(0xF870F8);
```

Comment:
I suggest to change the naming consistently to `kTrackColor<color name>`. This would align it with the enum class variants above, i.e. `TrackColor::<color name>`